src/pinns_pde_solve.jl

```diff
@@ -324,6 +328,30 @@ function build_symbolic_loss_function(eqs,indvars,depvars,
         vars_phi = Expr(:(=), build_expr(:tuple, phi_nums), build_expr(:tuple, expr_phi))
         push!(ex.args,  vars_phi)
     end
+    #Add an expression for paramater symbols
```

Comment:
Minor typo: "paramater" should be "parameter"

Reply:
Use the github suggest feature for typos.

```suggestion
    #Add an expression for parameter symbols
```